adding cleansing prior each application test

mmartin24 opened the pull request:
Linked to issue [#161 ](https://github.com/epinio/epinio-end-to-end-tests/issues/161)
The purpose is to clean applications from `applications.spec.ts` at the beginning of each test.
I assumed we will use all the time "testapp" as based.
If we want generic, it needs to be further implemented

juadk commented:
It could be useful to also check/remove `configuration01` (custom configuration name we use) and `mynamespace` (custom namespace we use).

mmartin24 commented:
> It could be useful to also check/remove `configuration01` (custom configuration name we use) and `mynamespace` (custom namespace we use).

I added cleanse of configuration as well on `applications.spec.ts`. I wonder if this cleanse should be added as well to `with_default_options.spec.ts`. If this is the case, perhaps the cleanse mechanism should be added in `tests.ts` instead of within the `integration` folder. 

When I added cleanse for namespaces I was not able to re-run the test without manually adding them. Also I had issues removing the configuration if namespace was `mynamespace` instead of `workspace`.

juadk reviewed (approved):
LGTM
